R/plotconfiguration-axis.R

```diff
@@ -287,7 +287,8 @@ XAxisConfiguration <- R6::R6Class(
             trans = self$ggplotScale(),
             breaks = self$prettyTicks(),
             labels = self$prettyTickLabels(),
-            expand = self$ggplotExpansion()
+            expand = self$ggplotExpansion(),
+            oob = .removeInfiniteValues
```

Comment:
`oob = .removeInfiniteValues` or
`oob = .removeInfiniteValues() `?

Reply:
`oob` argument expects a `function`. So, `oob = .removeInfiniteValues` is good
> Another way of doing this could be `match.fun(".removeInfiniteValues ")`

The default value is `oob = censor`, passing the function `censor` from the `{scales}` package. 
However , the function does not remove `Inf` values by default.

Interestingly, all the arguments of `scale_y_continous` function handle `function` type values.
This was actually the approach for solving the ticks and tick labels of log scale plots 
> `labels = self$prettyTickLabels()` returns functions such as `getLogTickLabels` for log scale plots, but can also return `getPiTickLabels` if you are working with angles in radians or `getGreekTickLabels` if you want to use greek letters to name indices (these functions were refined in PR #304 and we could also envision a function for setting a calendar time format for tick labels)

Reply:
interesting :)